config/grpc_client_gen.config

```diff
@@ -0,0 +1,34 @@
+{plugins, [
```

Comment:
note: client config only required if you plan on adding grpc tests and need to execute client behaviour

Reply:
Removed with commit c300421